build/images/deps/bgp-version

```diff
@@ -0,0 +1 @@
+3.28.0
```

Comment:
let's avoid having to maintain this
what I would suggest is to run `go list -m all | grep github.com/osrg/gobgp/v3 | cut -d " " -f2` as part of the docker build, in order to retrieve the version of gobgp that we currently import. This way we can keep the 2 versions (module version and binary version) consistent.